src/Symfony/Component/Form/Extension/Core/Type/WeekType.php

```diff
@@ -155,6 +155,11 @@ public function configureOptions(OptionsResolver $resolver)
             },
             'compound' => $compound,
             'choice_translation_domain' => false,
+            'invalid_message' => function (Options $options, $previousValue) {
+                return ($options['legacy_error_messages'] ?? true)
```

Comment:
on one line please

Reply:
This is copy-pasted from https://github.com/symfony/symfony/pull/30931/files#diff-9c4249df096b31311c1569e2328b1fe154f16d76d2041c48c51e3c01a0add742 Do you want me to update all form types?

Reply:
No need, that'd make merges more difficult.